packages/adapter-cloudflare/README.md

```diff
@@ -0,0 +1,59 @@
+# adapter-cloudflare
+
+[Adapter](https://kit.svelte.dev/docs#adapters) for building SvelteKit
+applications on Cloudflare Pages with Workers integration.
+
+_**Comparisons**_
+
```

Comment:
It might be nice to add a bit more detail here. E.g. I think it was mentioned that this should be the preferred one most of the time and only if you need to do something very advanced you would use `adapter-cloudflare-workers`? And it'd be nice to add that to the `adapter-cloudflare-workers` readme as well

Reply:
We can sunset the Workers one once the "Fullstack Pages" feature is out of this early beta phase. Should be in the next few weeks, which means I'll likely have an update for this by next meeting